pkg/cluster/meta/server_config.go

```diff
@@ -37,10 +37,8 @@ const (
 	migrateLockName = "tiup-migrate.lck"
 )
 
-// ConfigCheckError represent error occured in config check stage
-type ConfigCheckError struct {
-	error
-}
+// ErrorCheckConfig represent error occured in config check stage
+var ErrorCheckConfig = errors.New("check config failed")
```

Comment:
Use standard library `errors` package instead of `github.com/pingcap/errors` to prevent add stack information to the raw message.